### ISSUE TITLE
make underscore's source es6 friendly

### ISSUE THREAD

**Issue by tnga** [jashkenas/underscore#2557]
I plan to start working on underscore's source port to es6 usage. The goal is to have something like:

``` js
// full importation
import * as _ from 'underscore';

// or
// partial importation
import {_base} as _ from 'underscore';
import {isElement} as _.isElement from 'underscore';
```

This is firstly for some of my projects and i'am still thinking the best approach.

Is there actually a works or planifications for that ?

**Comment by akre54** [comment]
You can [already](http://babeljs.io/repl/#?evaluate=false&lineWrap=false&presets=es2015%2Creact%2Cstage-1%2Cstage-2%2Cstage-3&experimental=false&loose=false&spec=false&code=import%20%7B%20map%2C%20filter%20%7D%20from%20'underscore'%0A%0Amap(%5B1%2C%202%2C%203%5D%2C%20num%20%3D%3E%20num%20*%202)) [do that](http://babeljs.io/repl/#?evaluate=false&lineWrap=false&presets=es2015%2Creact%2Cstage-1%2Cstage-2%2Cstage-3&experimental=false&loose=false&spec=false&code=import%20*%20as%20_%20from%20'underscore'%0A%0A_.map(%5B1%2C%202%2C%203%5D%2C%20num%20%3D%3E%20num%20*%202)). 

We won't be supporting ES6 module exports in Underscore source for some time though.

**Comment by nebulousdog** [comment]
would be nice to have module exports

**Comment by mesqueeb** [comment]
How about now?

**Comment by jashkenas** [comment]
Maybe now is a good time?

**Comment by tnga** [comment]
Following can be a proposal since it have been release few days after have opened this issue.

https://github.com/tnga/underscore-es

**Comment by mhkeller** [comment]
Is this still something planned?

**Comment by jgonggrijp** [comment]
@mhkeller This is already implemented. See #2826, which was released with version 1.10.

Thanks for making me aware of this ticket, closing now.

**Comment by mhkeller** [comment]
Ah so cool! Thanks @jgonggrijp!
